### PR TITLE
pkg/loader: Refactor and add tests for executor

### PR DESCRIPTION
**Pull request by suzaku**
### What problem does this PR solve? 

[Jira Issue](https://internal.pingcap.net/jira/browse/TOOL-1050)

### What is changed and how it works?

Improve test coverage: 45.6% -> 61%

### Check List 

Tests 

 - Unit test

Code changes

Side effects

Related changes

### CODE REVIEW

File: pkg/loader/executor.go [hunk 1]
```diff
@@ -146,12 +150,11 @@ func (e *executor) bulkReplace(inserts []*DML) error {
...
 	info := inserts[0].info
-	dbName := inserts[0].Database
-	tableName := inserts[0].Table
 
-	builder := new(strings.Builder)
+	var builder strings.Builder
 
-	fmt.Fprintf(builder, "REPLACE INTO %s(%s) VALUES ", quoteSchema(dbName, tableName), buildColumnList(info.columns))
+	cols := buildColumnList(info.columns)
+	builder.WriteString(fmt.Sprintf("REPLACE INTO %s(%s) VALUES ", inserts[0].TableName(), cols))
```
Comment (kennytm) [inline pkg/loader/executor.go]: Same comment as https://github.com/pingcap/tidb-binlog/pull/579#discussion_r281110602. If you think that's not a problem then LGTM.

Comment (suzaku) [reply pkg/loader/executor.go]: Updated, PTAL.

**Comment by suzaku** [comment]
@kennytm conflict resolved.

**Pull request merged**